atest/testdata/standard_libraries/builtin/run_keyword_and_continue_on_failure.robot

```diff
@@ -57,6 +57,15 @@ Run Keyword And Continue On Failure With Timeout
     Run keyword And Continue On Failure    Sleep    1 second
     Fail    This Should Not Be Executed!
 
+Run Keyword And Continue On Failure With Nonexisting Variable
+    [Documentation]    FAIL   Variable '\${nonexisting}' not found.
+    Run keyword And Continue On Failure    Log    ${nonexisting}
+
+Run Keyword And Continue On Failure With Nonexisting Extended Variable
+    [Documentation]    FAIL   Resolving variable '\${list[2]}' failed: IndexError: list index out of range
+    ${list} =  Create list    1    2
+    Run keyword And Continue On Failure    Log  ${list[2]}
```

Comment:
With these tests it's not possible to know did the failure occur because the error escaped `Run Keyword And Continue On Failure` or because test ended and the continued failure was the only error that occurred. You could, for example, add something like `Fail    Normal failure after continuable failure` as the second step in these tests.